wallet-new/server/Cardano/Wallet/API/V1/Handlers/Transactions.hs

```diff
@@ -25,22 +26,23 @@ import           Test.QuickCheck (arbitrary, generate)
 handlers :: ( HasConfigurations
             , HasCompileInfo
             )
-         => ServerT Transactions.API MonadV1
+         => (TxAux -> MonadV1 Bool) -> ServerT Transactions.API MonadV1
 
-handlers = newTransaction
+handlers submitTx =
+             newTransaction submitTx
         :<|> allTransactions
         :<|> estimateFees
 
 newTransaction
     :: forall ctx m . (V0.MonadWalletTxFull ctx m)
-    => Payment -> m (WalletResponse Transaction)
-newTransaction Payment {..} = do
+    => (TxAux -> m Bool) -> Payment -> m (WalletResponse Transaction)
```

Comment:
This is suspicious, but `wallet-new` isn't well-maintained, so nvm.

Reply:
What's suspicious? I'd sure hope `wallet-new` is well maintained...